StratisCore.UI/src/app/shared/services/features.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { ApiService } from './api.service';
+
+@Injectable()
+export class FeaturesService {
+    private enabledFeatures: string[];
+
+    constructor(private apiService: ApiService) { }
+
+    getEnabledFeatures(): Observable<string[]> {
+        return this.enabledFeatures
+            ? Observable.of(this.enabledFeatures)
+            : this.apiService.getNodeStatus().map(x => x.enabledFeatures)
```

Comment:
for next time: 
1. you also need to figure out on what port to call the api. For stratis it's 37221, for a sidechain it might be 42555
2. you also need to figure out the right executable to run. For stratis it's located at `resources//daemon//Stratis.StratisD` and for sidechain it might be `resources//daemon//Stratis.SidechainNameD`

Reply:
Yeah, another subject.